content/_index.id.md

```diff
@@ -13,17 +13,15 @@ title: 'Organic Maps: Pendakian, Bersepeda, Jalan Setapak, dan Navigasi Luring'
 
 ### Unduh dan instal Organic Maps dari [AppStore][appstore], [Google Play][googleplay], [FDroid][fdroid], [Huawei AppGallery][appgallery] {#install}
 
-{{ badges(lang='id')}}}
+{{ badges(lang='id') }}
```

Comment:
Indonesian should be changed in .po files / weblate. Otherwise it will be overwritten. I am addin this missing spaces in .po and pushing this patch.